packages/turf-helpers/index.ts

```diff
@@ -57,15 +57,17 @@ export type AllGeoJSON =
  */
 
 /**
- * Earth Radius used with the Harvesine formula and approximates using a spherical (non-ellipsoid) Earth.
- *
+ * The {@link https://en.wikipedia.org/wiki/Earth_radius#Arithmetic_mean_radius arithmetic mean radius} of the Earth.
+ * Used by Turf modules that model the Earth as a sphere. This radii value
+ * was chosen over others because it is {@link https://rosettacode.org/wiki/Haversine_formula#:~:text=This%20value%20is%20recommended recommended for use} with the Haversine formula,
+ * used by turf/distance.
```

Comment:
I think this is useful background.  The intention is to follow this with #1176 and allow users to specify a custom earth radius in modules that support it (area, distance, etc.) similar to OpenLayers.  This will just become the default earthRadius.